dll/win32/winmm/playsound.c

```diff
@@ -121,15 +135,16 @@ static HMMIO	get_mmioFromProfile(UINT uFlags, LPCWSTR lpszName)
     err = RegOpenKeyW(hRegApp, lpszName, &hScheme);
     RegCloseKey(hRegApp);
     if (err != 0) goto none;
-    /* what's the difference between .Current and .Default ? */
```

Comment:
Isn't this question still valid?

Reply:
Based on my analysis, .Default means "Windows Default" profile as seen in mmsys.cpl and has nothing to do with the `PlaySound` function.

Reply:
(To clarify it's about the `.Current` vs. `.Default` values, the one selected in the screenshot below)

So your hypothesis, is that, for each windows event, (e.g. the "WindowsLogon"), there are the different sound profiles (e.g. "Afternoon", "Quirky" etc.), as well as the default Windows profile that would be that `.Default` one?
And so, would the value inside `.Current` reflect whatever value corresponding to the selected profile (or a custom sound if the user changed it via the mmsys.cpl GUI) ?

![image](https://user-images.githubusercontent.com/1969829/188700404-9080246c-96ba-41f0-a87e-12a6a7a751b3.png)

Reply:
![win_sounds](https://user-images.githubusercontent.com/86486978/188700882-03ffddbb-d112-4bb5-966f-4f7f508578a4.PNG)

This is how Windows store the sound profiles.

In the example, I created a new profile "ReactOS".

Reply:
OK that's what I understand. (Unrelated: dunno how you created the new profile, but it seems there's an off-by-one bug in the string length, and it shows up some garbage/the NULL terminator.)

Reply:
> would the value inside .Current reflect whatever value corresponding to the selected profile (or a custom sound if the user changed it via the mmsys.cpl GUI)

Yes, that's what I found.

> dunno how you created the new profile

Using Control Panel (mmsys.cpl) on WinXP.

> but it seems there's an off-by-one bug in the string length, and it shows up some garbage/the NULL terminator

That may be Windows' bug.